packages/models/project.json

```diff
@@ -46,6 +46,9 @@
       "cache": true,
       "inputs": ["production", "^production", "{projectRoot}/zod2md.config.ts"],
       "outputs": ["{projectRoot}/packages/models/docs/models-reference.md"]
+    },
+    "code-pushup": {
+      "executor": "@code-pushup/nx-plugin:autorun"
     }
```

Comment:
Same here, per-project targets are part of #706. I see you only added it for `models`, so I guess it's leftover from testing?

Reply:
Actually I thought I can keep this one in as "test". Will remove it and add it to #706

Reply:
removed it